config/environments/default.py

```diff
@@ -0,0 +1,102 @@
+"""Flask configuration."""
+from os import environ
+from os import path
+
+
+class Config(object):
```

Comment:
📌  I think that we are reaching a critical point in our configuration where a new solution is needed.